src/components/App.tsx

```diff
@@ -11,9 +11,11 @@ import { useMegaStore } from "~/state/megaStore";
 import { Show } from "solid-js";
 import { ExternalLink } from "./layout/ExternalLink";
 import { BetaWarningModal } from "~/components/BetaWarningModal";
+import {useI18n} from "../i18n/context";
```

Comment:
can use the `~` alias like this: `~/i18n/context` I know I'm not super consistent with it but it's nice especially for something like this that's gonna get imported all over the place

Reply:
Fixed. Marking this conversation as resolved.